Src/Doxyfile

```diff
@@ -1061,9 +1089,6 @@ EXCLUDE_PATTERNS       =
 # output. The symbol name can be a fully qualified name, a word, or if the
 # wildcard * is used, a substring. Examples: ANamespace, AClass,
 # ANamespace::AClass, ANamespace::*Test
-#
-# Note that the wildcards are matched against the file with absolute path, so to
-# exclude all test directories use the pattern */test/*
```

Comment:
Is this comment no longer relevant?

Reply:
doxygen has an option to "update a doxygen-file to the latest version" ('-u' I guess, c.f. [here](https://man.archlinux.org/man/doxygen.1.en)). This is what I ran here, and it updated the comments to the latest version.